lib/usdUfe/ufe/UsdClipboardHandler.cpp

```diff
@@ -66,8 +69,30 @@ UsdClipboardHandler::Ptr UsdClipboardHandler::create()
 
 Ufe::UndoableCommand::Ptr UsdClipboardHandler::cutCmd_(const Ufe::Selection& selection)
 {
+    // Don't allow cutting (which also means copying) the items which cannot
+    // be cut.
+    Ufe::Selection allowedToBeCut;
+    for (const auto& item : selection) {
+        // EMSUSD-1126 - Cut a prim and not have it paste if the cut is restricted
+        auto       usdItem = downcast(item);
+        const auto prim = usdItem ? usdItem->prim() : PXR_NS::UsdPrim();
+        if (!prim)
+            continue;
+
+        // As per the JIRA item for now, skip the special cut conditions
+        // on shaders and nodegraphs. These nodes are handled by special
+        // cases in LookdevX plugin.
+        if ((PXR_NS::UsdShadeNodeGraph(prim) || PXR_NS::UsdShadeShader(prim))
+            || canBeCut_(item)) {
+            allowedToBeCut.append(item);
+        }
+    }
+
+    if (allowedToBeCut.empty())
+        return {};
```

Comment:
Implement the cut restrictions (except on node graph and shaders, as directed in JIRA ticket).